compiler/dyno/include/chpl/queries/Context.h

```diff
@@ -102,9 +103,27 @@ class Context {
   bool breakSet = false;
   size_t breakOnHash = 0;
   int numQueriesRunThisRevision_ = 0;
+  // tracks the nesting of queries, displayed during query tracing
+  int queryTraceDepth = 0;
+  // list of query names to ignore when tracing
+  const std::vector<std::string> queryTraceIgnoreQueries = {"idToTagQuery", "idToParentId"};
```

Comment:
I think it would be cool to have this be configurable in the future (I'm sure you already have plans for this).

Reply:
we briefly talked about doing something like that, for now the expectation is that you recompile to adjust this list